test/libsolidity/syntaxTests/functionCalls/external_functions_with_calldata_args_assgined_to_function_pointers_of_memory_type.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
```

Comment:
Typo in test name: `assgined` -> `assigned`.

Also `function_pointers_of_memory_type` -> `function_pointer_with_memory_args`.